server/webapp/WEB-INF/rails/webpack/views/components/table/index.scss

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import "../../global/common";
+
+$table-border: $line-color;
+$table-header-bg: $secondary-bg;
+
+.table-component {
+  $table-border-top-color: #dee2e6;
+  border:          1px solid $table-border;
+  border-collapse: collapse;
+  display:         table;
+  width:           100%;
+  font-family: "Open Sans", sans-serif;
```

Comment:
Indent file.

Did font-family need to be defined here? I'd think there's a global font-family somewhere?

Reply:
Removed explicit font-family declaration as it is coming from global.